config/routes.rb

```diff
@@ -1,6 +1,8 @@
 require 'api_constraints'
 
 Foreman::Application.routes.draw do
+  use_doorkeeper
```

Comment:
This added the following routes

GET       /oauth/authorize/:code  (Not using)
GET       /oauth/authorize (Not using)
POST      /oauth/authorize (Not using)
PUT       /oauth/authorize (Not using)
DELETE    /oauth/authorize (Not using)
POST      /oauth/token (USING - most important route)
POST      /oauth/revoke
resources /oauth/applications 
GET       /oauth/authorized_applications (works, restricted to admins, but not currently needed)
DELETE    /oauth/authorized_applications/:id  (works)
GET       /oauth/token/info  (works)